src/vs/editor/contrib/codeAction/browser/lightBulbWidget.css

```diff
@@ -23,6 +23,11 @@
 	color: var(--vscode-editorLightBulbAutoFix-foreground, var(--vscode-editorLightBulb-foreground));
 }
 
+.monaco-editor .lightBulbWidget.codicon-sparkle-filled {
+	color: var(--vscode-icon-foreground);
+	opacity: 0.5;
+}
```

Comment:
I think a better fix would be the following

1. define a new color [here](https://github.com/microsoft/vscode/blob/028e94157e76c17382d0fd8b3431433f9ee67bac/src/vs/platform/theme/common/colorRegistry.ts#L402) (the color should inherit from an existing color but make it lighter) 
3. use the new color (like [so](https://github.com/microsoft/vscode/blob/09020f55c0b3559f3de2b8d1dea15eeb2ba90b3f/src/vs/editor/contrib/codeAction/browser/lightBulbWidget.css#L18)) instead of the generic color